src/azure-cli/azure/cli/command_modules/sqlvm/custom.py

```diff
@@ -127,13 +127,10 @@ def sqlvm_group_update(instance, domain_fqdn=None, cluster_operator_account=None
 def sqlvm_aglistener_create(client, cmd, availability_group_listener_name, sql_virtual_machine_group_name,
                             resource_group_name, availability_group_name, ip_address, subnet_resource_id,
                             load_balancer_resource_id, probe_port, sql_virtual_machine_instances, port=1433,
-                            public_ip_address_resource_id=None, vnet_name=None):
+                            public_ip_address_resource_id=None, vnet_name=None):  # pylint: disable=unused-argument
```

Comment:
If `vnet_name` is not used, can we remove it?

Reply:
If we remove it, it will be a break change to the upper-level command `az sql vm group ag-listener create` as `--vnet-name` will be removed. I only fix the linter error, so prefer not to change it.